### DISCUSSION TITLE
IPV6 Add The server

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#2219] [Development discussions]
Hi All

can you please help me how to add Ipv6 on postal i am try to add but i can't, can you please help me 

![image](https://user-images.githubusercontent.com/107031529/199791808-466977e5-471d-46d4-99d4-443810c2c46a.png)

**Comment by willpower232** [comment]
Has your server provider definitely not given you an IPv4 address as well as an IPv6 one?

**Reply by arunkuttytry** [reply]
I have IPv4 and multiple IPv6, how can I configure IPv6?

**Reply by willpower232** [reply]
It depends how they are related to each other, if you only have one IPv4 then I think you're going to struggle to use Postal with IP Pools.

**Comment by arunkuttytry** [comment]
Hi

I have IPv4 and multiple IPv6, how can I configure IPv6?
